tutorials/migration.md

```diff
@@ -22,178 +25,122 @@ This page is structured as follows:
 
-The following procedure primarily focuses on an in-place migration. If you are
-migrating to a fresh install, container, or another VM some of the following
-steps can be skipped. Those steps are identified.
+The following procedure primarily focuses on an in-place migration. 
+If you are migrating to a fresh install, container, or another VM some of the following steps can be skipped. 
+Those steps are identified.
 
-One major consideration is that at the time of this writing openHAB 2 does not
-implement authentication and authorization (i.e. no username and password). If
-you are relying upon port forwarding to access your openHAB server remotely
-instead of via a VPN, SSH tunneling, or [my.openhab](https://my.openhab.org/),
-we highly recommend [setting up a reverse proxy]({{base}}/configuration/nginx.html).
+One major consideration is that at the time of this writing openHAB 2 does not implement authentication and authorization (i.e. no username and password). 
+If you are relying upon port forwarding to access your openHAB server remotely instead of via a VPN, SSH tunneling, or [my.openhab](https://my.openhab.org/), we highly recommend [setting up a reverse proxy]({{base}}/configuration/nginx.html).
 
 ## Backups
 
-The first step is to backup everything that you have modified in the existing
-openHAB 1.x installation. If you installed using apt-get these files may include:
+The first step is to backup everything that you have modified in the existing openHAB 1.x installation. 
+If you installed using apt-get these files may include:
 
 - `/etc/openhab` - configuration files
-- `/var/lib/openhab` - openhab user's home directory, zwave and embedded
-persistence database files
-- `/usr/share/openhab/webapps` - custom icons, custom webviews (e.g.
-[Weather Binding](https://github.com/openhab/openhab/wiki/Weather-Binding))
-- `/usr/share/openhab/bin` - custom command line arguments to the Java Virtual
-Machine, changes to the default ports, etc.
+- `/var/lib/openhab` - openhab user's home directory, zwave and embedded persistence database files
+- `/usr/share/openhab/webapps` - custom icons, custom webviews (e.g. [Weather Binding](https://github.com/openhab/openhab/wiki/Weather-Binding))
+- `/usr/share/openhab/bin` - custom command line arguments to the Java Virtual Machine, changes to the default ports, etc.
 - `/etc/default/openhab` - an alternative place to change default ports
 
-If you are on an operating system that does not support apt-get or performed a
-manual installation for some other reason simply backup the whole
-`<openhab home>` directory.
+If you are on an operating system that does not support apt-get or performed a manual installation for some other reason simply backup the whole `<openhab home>` directory.
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.
See https://github.com/openhab/openhab-docs/issues/189#issuecomment-360202149 for an overview and a link where to find the source.

So in this case `$OPENHAB_HOME` would be the right choice.